src/main/java/imsam/mgen/MultiTargetModelGenerator.java

```diff
@@ -0,0 +1,347 @@
+package imsam.mgen;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.json.JSONObject;
+import org.kohsuke.args4j.Option;
+
+
+/**
+ * This is a model generator that will accept a list of target states and force all paths to converge on one of said states
+ * @author Eric Reiss
+ */
+public class MultiTargetModelGenerator extends MGen{
+    public List<Integer> targets;
+    public static final String MGEN_ID = "multi-target";
+    public int[][] adjacencyMatrix; //Matrix to draw graph with 
+
+    //////////////////////////////////////////////////
+    // CLI Arguments
+    @Option(name="--target-list",usage="In a space-separated String, list states that all paths will converge to. default: generates random targets")
+    public String targetList = "";
+    // end CLI Arguments
+    //////////////////////////////////////////////////
+
+
+    //Set default file name
+    @Override
+    public String MGEN_ID(){
```

Comment:
It's generally bad practice to have a function with the same name as a data member, even if your compiler allows it. I don't think that it will compile, though.

Reply:
Renamed to MODEL_ID

Reply:
Might be better renaming the method as `getMGenID()` and the variable stays `MGEN_ID`

Reply:
Make these changes to `SparseModelGenerator` as well